src/main/python/org/broadinstitute/hellbender/gcnvkernel/io/io_denoising_calling.py

```diff
@@ -155,7 +157,8 @@ def __call__(self):
         # compute approximate denoised copy ratios
         _logger.info("Sampling and approximating posteriors for denoised copy ratios...")
         denoising_copy_ratios_st_approx_generator = commons.get_sampling_generator_for_model_approximation(
-            model_approx=self.denoising_model_approx, node=self.denoising_model['denoised_copy_ratio_st'])
+            model_approx=self.denoising_model_approx, node=self.denoising_model['denoised_copy_ratio_st'],
+            num_samples=self.num_samples_for_cr_approx)
```

Comment:
Probably OK to have `num_samples_for_cr_approx` differ from the Java names here, but I might just make them all consistent. Might also change `num_samples` -> `num_samples_for_cr_approx`, since it's otherwise ambiguous (i.e., sequenced samples vs. posterior samples).

Reply:
`num_samples` is a parameter of the method that can sample from any model nodes not just CR, I updated all the other names however